src/api/v1/jobs/controller.ts

```diff
@@ -6,17 +6,17 @@ import type { TypedRequestHandlers } from '@openapi';
 import { SERVICES, successMessages } from '@common/constants';
 import { SamePriorityChangeError } from '@src/jobs/models/errors';
 import { IllegalJobStatusTransitionError, JobNotInFiniteStateError, JobNotFoundError } from '@src/common/generated/errors';
-import { JobManager } from '../models/manager';
-import { type JobFindCriteriaArg } from '../models/models';
+import { JobManager } from '../../../jobs/models/manager';
+import { type JobFindCriteriaArg } from '../../../jobs/models/models';
 
 @injectable()
-export class JobController {
+export class JobControllerV1 {
```

Comment:
i wonder if you actually need V1 in the name, as i dont think its imported outside of the v1 folder